client/service/src/benchmark.rs

```diff
@@ -0,0 +1,69 @@
+#![cfg(feature = "rocksdb")]
```

Comment:
This whole file and the whole benchmark command doesn't work without the feature rocksdb in sc_service. So I added the same rocksdb flag for sc_cli so I can compile it alone (with or without rocksdb) and it doesn't break.

Reply:
This whole benchmark command was moved to its own crate `utils/frame/benchmarking-cli`